modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ddl/DdlCommandHandlerWrapper.java

```diff
@@ -53,8 +57,15 @@ public DdlCommandHandlerWrapper(
     @Override
     public CompletableFuture<Boolean> handle(DdlCommand cmd) {
         if (cmd instanceof CreateTableCommand) {
```

Comment:
since we always have to call super, does it make sense to move `if` statement inside `thenCompose`?